src/lib.rs

```diff
@@ -20,25 +20,25 @@ create_exception!(_rtoml, TomlParsingError, PyValueError);
 create_exception!(_rtoml, TomlSerializationError, PyValueError);
 
 #[pyfunction]
-fn deserialize(py: Python, toml_data: String) -> PyResult<PyObject> {
+fn deserialize(py: Python, toml_data: String, none_value: Option<&str>) -> PyResult<PyObject> {
     let mut deserializer = Deserializer::new(&toml_data);
-    let seed = de::PyDeserializer::new(py);
+    let seed = de::PyDeserializer::new(py, none_value);
     seed.deserialize(&mut deserializer)
         .map_err(|e| TomlParsingError::new_err(e.to_string()))
 }
 
 #[pyfunction]
-fn serialize(py: Python, obj: &PyAny) -> PyResult<String> {
-    let s = SerializePyObject::new(py, obj);
+fn serialize(py: Python, obj: &PyAny, none_value: String) -> PyResult<String> {
+    let s = SerializePyObject::new(py, obj, &none_value);
```

Comment:
```suggestion
fn serialize(py: Python, obj: &PyAny, none_value: &str) -> PyResult<String> {
    let s = SerializePyObject::new(py, obj, none_value);
```

Hopefully avoid one allocation.